dep/Console/condrv.h

```diff
@@ -158,7 +158,7 @@ typedef struct _CD_IO_DISPLAY_SIZE {
 
 typedef struct _CD_IO_CHARACTER {
     WCHAR Character; 
-    USHORT Atribute;
+    USHORT Attribute;

```

Comment:
This file is in a directory `dep/` which probably means I shouldn't be touching it. Just let me know, and I'll drop it/all similar items.

If there's an upstream that I can reach, at some point I can provide a similar PR to it, if it's identified here, I'll probably get to it sooner.

Reply:
I've dropped the `dep/` dirs